### PR TITLE
Added appveyor support, #152

### LINKED ISSUE #152: Use windows CI as well
@lk-geimfari can you please help me with this one?

I have never tried to enable windows CI and I am pretty sure it will have issues.
But, you somehow managed to get this working with `mimesis`.

### PR DESCRIPTION
**Pull request by lk-geimfari**
Fixed #152

### CODE REVIEW

**Comment by lk-geimfari** [comment]
@sobolevn So, now we need to just add this repository to AppVeyor account.

**Comment by lk-geimfari** [comment]
Do you have an account on AppVeyor?

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  - "poetry develop"
+
+build: off
+
+test_script:
+  - "poetry run pytest"
+  - "poetry run mypy wemake_python_styleguide"
+  - "poetry run doc8 -q docs"
+
+on_success:
```
Comment (lk-geimfari) [inline appveyor.yml]: I'm not sure about this. Do we really need to check it in both CI- on Travis and AppVeyor?

Comment (sobolevn) [reply appveyor.yml]: Now, let's leave it to travis.

**Comment by lk-geimfari** [comment]
@sobolevn I have tried to build it on my account, but seems like i have not needed permissions to repository. Can you, please, add this project on your own AppVeyor account?

**Comment by sobolevn** [comment]
AppVeyor url: https://ci.appveyor.com/project/sobolevn/wemake-python-styleguide

**Comment by sobolevn** [comment]
I guess it can be easier to create a new PR if the integration will not work for this one.

**Pull request closed without merging**